databases/week3/meal_application.sql

```diff
@@ -0,0 +1,33 @@
+CREATE DATABASE meal_application;
+USE meal_application;
+
+CREATE TABLE `Meal` (
+  `id` int,
+  `title` varchar(255),
+  `description` text,
+  `location` varchar(255),
+  `when` datetime,
+  `max_reservations` int,
+  `price` decimal,
+  `created_date` date
+);
+
+CREATE TABLE `Reservation` (
+  `id` int,
+  `number_of_guests` int,
+  `meal_id` int,
+  `created_date` date
+);
+
+CREATE TABLE `Review` (
+  `id` int,
```

Comment:
`id` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,

Reply:
Fixed. Thank you